dali/python/nvidia/dali/_debug_mode.py

```diff
@@ -23,12 +23,13 @@
 import nvidia.dali.pipeline as _pipeline
 import nvidia.dali.tensors as _tensors
 import nvidia.dali.types as _types
+from nvidia.dali import _conditionals
 from nvidia.dali._utils.eager_utils import _Classification, _transform_data_to_tensorlist
 from nvidia.dali.data_node import DataNode as _DataNode, _check
 from nvidia.dali.fn import _to_snake_case
-from nvidia.dali._utils.external_source_impl import (
-    get_callback_from_source as _get_callback_from_source,
-    accepted_arg_count as _accepted_arg_count)
+from nvidia.dali._utils.external_source_impl import (get_callback_from_source as
+                                                     _get_callback_from_source, accepted_arg_count
+                                                     as _accepted_arg_count)
```

Comment:
Sorry, but this is really eye-hostile.
```suggestion
                                                     _get_callback_from_source,
                                                     accepted_arg_count as _accepted_arg_count)
```